### DISCUSSION TITLE
Allowing multiple table search in a row

### DISCUSSION THREAD

**Discussion by prawn969** [beekeeper-studio/beekeeper-studio#1798] [Ideas]
Hi,
It would be awesome if we could search more than just one table at a time with a REGEX for example, or just separate texts with a | (pipe).
![image](https://github.com/beekeeper-studio/beekeeper-studio/assets/8478915/674addc3-f857-4e8f-8afa-d6da4780746d)

**Comment by troyporter** [comment]
I came here to post this exact thing!

HeidiSQL has this feature and it is **VERY** useful.  I hope the BeeKeeper team looks into this.
